src/helper.cc

```diff
@@ -1439,14 +1434,14 @@ helperDispatch(helper_server * srv, Helper::Xaction * r)
     }
 
     r->request.Id = reqId;
-    helper_server::Requests::iterator it = srv->requests.insert(srv->requests.end(), r);
+    Helper::Session::Requests::iterator it = srv->requests.insert(srv->requests.end(), r);
```

Comment:
Can this use `auto` instead?
```suggestion
    const auto it = srv->requests.insert(srv->requests.end(), r);
```
or 
```suggestion
    auto it = srv->requests.insert(srv->requests.end(), r);
```